src/landing/LandingPage.js

```diff
@@ -62,7 +62,7 @@ const LandingPage = () => {
   const submitRegister = (firstName, lastName, username, password) => {
     createUser({
       variables: {
-        username, firstName, lastName, password,
+        username, firstName, lastName, password, userRole: 3,
```

Comment:
In cases like this, it is a good practice to user constants instead of what we call "magic numbers". 
IE const basicUser = 3;
userRole: basicUser. Doing this will make the code more readable.

Reply:
fixed